msal/src/Microsoft.Identity.Client/TokenCache.cs

```diff
@@ -276,7 +276,7 @@ private bool ContainsClaimsRequiredToStoreInCache(IdToken idToken, RequestContex
                 }
                 else
                 {
-                    msg = "Skipping storing tokens in the cache - IdToken does not contain required claims";
+                    msg = "Skipping storing tokens in the cache - IdToken does not contain required claims of TenantId or PreferredUsername.";
```

Comment:
>of TenantId or PreferredUsername [](start = 107, length = 32)

we should use the actual names used in the claims. ideally we have const somewhere with these in that we can use.

Reply:
done. thx. ⛄️